hpobench/benchmarks/mo/cnn_benchmark.py

```diff
@@ -0,0 +1,535 @@
+                'kernel_size', default_value=5, choices=[7, 5, 3]
+            )
+
+        ])
+        return cs
+
+    @staticmethod
+    def get_objectives():
+        return ['accuracy', 'model_size']
+
+    @staticmethod
+    def get_fidelity_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+
+        fidelity_space = CS.ConfigurationSpace(seed=seed)
+        fidelity_space.add_hyperparameters(
+            # gray-box setting (multi-multi-fidelity) - iterations + data subsample
+            CNNBenchmark._get_fidelity_choices(iter_choice='variable', subsample_choice='variable')
+        )
+        return fidelity_space
+
+    @staticmethod
+    def _get_fidelity_choices(iter_choice: str, subsample_choice: str) -> Tuple[Hyperparameter, Hyperparameter]:
+
+        fidelity1 = dict(
+            fixed=CS.Constant('budget', value=50),
+            variable=CS.UniformIntegerHyperparameter(
+                'budget', lower=1, upper=50, default_value=50, log=False
+            )
+        )
+        fidelity2 = dict(
+            fixed=CS.Constant('subsample', value=1),
+            variable=CS.UniformFloatHyperparameter(
+                'subsample', lower=0.1, upper=1, default_value=1, log=False
+            )
+        )
+        budget = fidelity1[iter_choice]
+        subsample = fidelity2[subsample_choice]
+        return budget, subsample
+
+    @staticmethod
+    def get_meta_information() -> Dict:
+        """ Returns the meta information for the benchmark """
+        return {
+            'name': 'Bag of baselines for multi-objective joint neural architecture search and hyperparameter optimization',
+            'references': ['@article{guerrero2021bag,'
+                           'title   = {Bag of baselines for multi - objective joint neural architecture search and hyperparameter optimization},'
+                           'author  = {Guerrero-Viu, Julia and Hauns, Sven and Izquierdo, Sergio and Miotto, Guilherme and Schrodi, Simon and Biedenkapp, Andre and Elsken, Thomas and Deng, Difan and Lindauer, Marius and Hutter, Frank},},'
+                           'journal = {arXiv preprint arXiv:2105.01015},'
+                           'year    = {2021}}',
+                           ],
+            'code': 'https://github.com/automl/multi-obj-baselines',
+        }
+
+    def init_model(self, config: Union[CS.Configuration, Dict],
+                   fidelity: Union[CS.Configuration, Dict, None] = None,
+                   rng: Union[int, np.random.RandomState, None] = None):
+        """ Function that returns the model initialized based on the configuration and fidelity
+        """
+        rng = self.rng if rng is None else rng
+
+        if isinstance(config, CS.Configuration):
+            config = config.get_dictionary()
+        if isinstance(fidelity, CS.Configuration):
+            fidelity = config.get_dictionary()
+        return Net(config, (3, 16, 16), 17)
+
+    @AbstractBenchmark.check_parameters
```

Comment:
Please use 

```suggestion
    @AbstractBenchmark.check_parameters
    @AbstractMultiObjectiveBenchmark.check_parameters
```
here. Also, please for the objective function test.